README.md

```diff
@@ -1,2 +1,81 @@
-# mi-vscode
-Micro Integrator extension for Visual Studio Code
+# The Micro Integrator extension for Visual Studio Code
```

Comment:
Micro Integrator for Visual Studio Code

Reply:
Fixed in https://github.com/wso2/mi-vscode/pull/1/commits/6c54f772dc0c4bb077bf9dc0491ebe02c827f53b.